boards/remote-pa/include/board.h

```diff
@@ -70,11 +70,10 @@
  * 2.4GHz RF switch controlled by SW
  * @{
  */
-#define RF_SWITCH_PORT      GPIO_D
-#define RF_SWITCH_PIN       (4)
-#define RF_SWITCH_EXTERNAL  (RF_SWITCH_PORT->DATA |= (1 << RF_SWITCH_PIN))
-#define RF_SWITCH_INTERNAL  (RF_SWITCH_PORT->DATA &= ~(1 << RF_SWITCH_PIN))
-#define RF_SWITCH_TOGGLE    (RF_SWITCH_PORT->DATA ^= (1 << RF_SWITCH_PIN))
+ #define RF_SWITCH_GPIO      GPIO_PD4
```

Comment:
yes, but there are lots of these macro usages throughout the cc2538 code base and I remove them in #7320, all at once.